flake.nix

```diff
@@ -106,22 +104,7 @@
         };
 
         devShells.default = pkgs.mkShell {
-          packages = with pkgs; [
-            rustc
-            cargo
-          ];
-
-          buildInputs = (lib.optionals pkgs.stdenv.isDarwin [
-            pkgs.darwin.apple_sdk.frameworks.CoreServices
-            pkgs.libiconv
-          ]) ++ [
-            pkgs.openssl
-          ];
-
-          nativeBuildInputs = [
-            pkgs.pkg-config
-            pkgs.openssl.dev
-          ];
+          inputsFrom = [ typst ];
```

Comment:
I noticed that this doesn't add `rustc` anymore. May be worth explicitly adding it back. `cargo` still seems to be there already, however.

(Unless my setup is somehow dirty - may be worth testing locally.)

Reply:
Tbh it may be worth adding both cargo and rustc explicitly regardless. We only need `inputsFrom = [ typst ];` to ensure we pull the native stuff anyway.